ui/src/message_popup/linear_session/record_then_classify_question.jsx

```diff
@@ -29,11 +30,27 @@ export default React.createClass({
   },
 
   onDoneAudio(audioResponse) {
-    this.setState({audioResponse});
+    if(this.props.question.choices.length > 0) {
```

Comment:
If there are no choices, this will change the behavior the component in a way that will affect the user experience in other pages, which I don't think we want.

Could you instead branch in `DansonExperiencePage2#renderQuestionEl` and check if there are choices?  If there are choices, use `RecordThenClassify` as it is now without any changes.  And if there are choices, use `MinimalOpenResponse` directly.  Would something like that help achieve what we want, without changing this component's behavior and affecting other pages?

Something like this is how [InsubordinationPage#renderQuestionEl](https://github.com/kesiena115/threeflows/blob/ddeaba5f45b122401ff8f65fdba6425b7c5382af/ui/src/message_popup/playtest/insubordination_page.jsx#L109) works.